src/main/scala/cromwell/engine/backend/local/LocalBackend.scala

```diff
@@ -163,12 +163,19 @@ class LocalBackend extends Backend with SharedFileSystem with LazyLogging {
     val stderrFileLength = Files.size(backendCall.stderr)
     val rc = Try(backendCall.rc.slurp.stripLineEnd.toInt)
     if (backendCall.call.failOnStderr && stderrFileLength > 0) {
-      Failure(new Throwable(s"Call ${backendCall.call.fullyQualifiedName}, Workflow ${backendCall.workflowDescriptor.id}: stderr has length $stderrFileLength"))
+      FailedExecution(new Throwable(s"Call ${backendCall.call.fullyQualifiedName}, Workflow ${backendCall.workflowDescriptor.id}: stderr has length $stderrFileLength"))
     } else {
       rc match {
-        case Success(0) => postProcess(backendCall)
-        case Success(143) => Failure(new TaskAbortedException()) // Special case to check for SIGTERM exit code - implying abort
-        case _ => Failure(new Throwable(s"Call ${backendCall.call.fullyQualifiedName}, Workflow ${backendCall.workflowDescriptor.id}: $rc\n\nFull command was: ${argv.mkString(" ")}"))
+        case Success(0) =>
+          postProcess(backendCall) match {
+            case Success(outputs) => SuccessfulExecution(outputs)
+            case Failure(e) =>
+              val message = Option(e.getMessage) map { ": " + _ } getOrElse ""
+              FailedExecution(new Throwable("Failed post processing of outputs" + message, e))
+          }
+        case Success(143) => AbortedExecution // Special case to check for SIGTERM exit code - implying abort
+        case Success(badRc) => FailedExecution(new Throwable(s"Call ${backendCall.call.fullyQualifiedName}, Workflow ${backendCall.workflowDescriptor.id}: $rc\n\nFull command was: ${argv.mkString(" ")}"), Option(badRc))
```

Comment:
I guess `exit` is a shell built-in which exits the current shell right away.  If I replaced `exit 1` with `python -c "import sys; sys.exit(1)"`, then it got to the `Success(badRc)` case.